src/components/NewDraftWizard/WizardForms/WizardFillObjectDetailsForm.js

```diff
@@ -432,20 +441,57 @@ const FormContent = ({ resolver, formSchema, onSubmit, objectType, folderId, cur
  */
 const WizardFillObjectDetailsForm = (): React$Element<typeof Container> => {
   const classes = useStyles()
+  const dispatch = useDispatch()
 
   const objectType = useSelector(state => state.objectType)
+  const { folderId } = useSelector(state => state.submissionFolder)
+  const currentObject = useSelector(state => state.currentObject)
+
+  // States that will update in useEffect()
+  const [states, setStates] = useState({
+    error: false,
+    errorPrefix: "",
+    formSchema: {},
+    validationSchema: {},
+    isLoading: true,
+  })
 
-  const [isLoading, setIsLoading] = useState(true)
-  const [error, setError] = useState(false)
-  const [errorPrefix, setErrorPrefix] = useState("")
   const [successStatus, setSuccessStatus] = useState("")
-  const [formSchema, setFormSchema] = useState({})
-  const [validationSchema, setValidationSchema] = useState({})
   const [submitting, setSubmitting] = useState(false)
-  const dispatch = useDispatch()
-  const { folderId } = useSelector(state => state.submissionFolder)
   const [responseInfo, setResponseInfo] = useState([])
-  const currentObject = useSelector(state => state.currentObject)
+
+  /*
+   * Fetch json schema from either local storage or API, set schema and dereferenced version to component state.
```

Comment:
```suggestion
   * Fetch json schema from either session storage or API, set schema and dereferenced version to component state.
```
nitpicking as we have not been using local storage for quite some time we should make it clear it is session storage